lib/gems/pending/appliance_console/cli.rb

```diff
@@ -286,6 +293,15 @@ def extauth_opts
       extauthopts.update_configuration(extauthopts_hash)
     end
 
+    def server_mode
+      service = LinuxAdmin::Service.new("evmserverd")
+      case server_state
```

Comment:
Shouldn't this be `options[:server]`?

I'm not sure where the `server_state` variable is coming from here.